### ISSUE TITLE
Feature Request: Using Villager Tools

### ISSUE THREAD

**Issue by MakaniValur** [TangoTek/TekTopia-Community#593]
It'd be lovely to give players the ability to help villagers out more actively. To avoid the balancing issues otherwise caused, the idea would be to let the player use villager tools. Either those that a villager has discarded because he got an upgrade or one that the blacksmith/enchanter have made. While using that tool, the player too can get villager items. If they try to improve the tool in any way (by enchanting it etc) it loses villager status. 

Might have to make it so crops can be broken with a hoe though. But that way players could get villager wood, mine for the villagers, collect milk, shear sheep and maybe even harvest crops.

Basically, it allows players to be more actively involved, especially early game.
